tests/test_bkz.cpp

```diff
@@ -14,6 +14,7 @@
    along with fplll. If not, see <http://www.gnu.org/licenses/>. */
 
 #include "io/json.hpp"
+#include <../tests/test_utils.h>
```

Comment:
I tests different paths (`test_utils.h`, `tests/test_utils.h`) that passes `make check` but not `make distcheck`. For the first one, CI reports

> `../../tests/test_lll.cpp:18:24: fatal error: test_utils.h: No such file or directory`
> `#include <test_utils.h>`

Then, I tried `../tests/test_utils.h` and it works for both `make check` and `make distcheck`. But I am also not really happy with this fix.